.claude/commands/status.md

```
@@ -0,0 +1,138 @@
+---
+description: 작업 진행률 시각화. design 체크리스트 대비 구현 완료 현황을 간단히 표시.
+tags: [project]
+---
+
+# /status - 진행률 확인
+
+작업 진행 현황을 빠르게 확인합니다.
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)